src/corona_hakab_model/common/util.py

```diff
@@ -25,7 +30,11 @@ def off_binom(a, c, b):
     if len(args) == 1:
         return const_dist(*args)
     if len(args) == 2:
-        return uniform_dist(*args)
+        if type(args[0]) == list and type(args[1]) == list:
+            assert len(args[0]) == len(args[1]), f"Elements and weights vectors should be in same size! (elements: {len(args[0])}, weights: {len(args[1])})"
```

Comment:
What if both are empty?

Reply:
added assert for checking that both are non-empty lists